src/util/azure/resource-group.ts

```diff
@@ -52,7 +52,7 @@ export async function getResourceGroup(
   spinner.stop();
   let result;
 
-  const initialName = options.project + '-static-deploy';
+  const initialName = `ngdeploy-${options.project}-cxa`;
```

Comment:
is the `cxa` suffix a requirement? Users will have this in their resource names.

Reply:
Trying to figure out a unique identifier :)